tests/model_explainability/guardrails/test_guardrails.py

```
@@ -589,3 +589,71 @@ def test_guardrails_autoconfig_gateway_negative_detection(
+@pytest.mark.usefixtures("patched_dsc_kserve_headed")
+class TestGuardrailsOrchestratorCustomTLS:
+    """
+    Tests custom TLS certificate mounting for the GuardrailsOrchestrator.
+    Verifies that custom TLS secrets specified in the CR are correctly mounted
+    to the orchestrator deployment at /etc/tls/$SECRET_NAME.
+    """
+
+    def test_custom_tls_secret_mounted(
+        self,
+        admin_client: DynamicClient,
+        model_namespace: Namespace,
```

Flake8 (7.3.0)

[error] undefined name 'Namespace'

(F821)